docs/en/get_started.md

```diff
@@ -0,0 +1,268 @@
+   ```
+
+3. Install mmcv-full, we recommend you to install the pre-build package as below.
+
+   ```shell
+   pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/{cu_version}/{torch_version}/index.html
+   ```
+
+   mmcv-full is only compiled on PyTorch 1.x.0 because the compatibility usually holds between 1.x.0 and 1.x.1. If your PyTorch version is 1.x.1, you can install mmcv-full compiled with PyTorch 1.x.0 and it usually works well.
+
+
+   ```shell
+   # We can ignore the micro version of PyTorch
+   pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu113/torch1.11.0/index.html
+   ```
+
+   See [here](https://mmcv.readthedocs.io/en/latest/get_started/installation.html) for different versions of MMCV compatible to different PyTorch and CUDA versions.
+   Optionally you can choose to compile mmcv from source by the following command
+
+   ```shell
+   git clone https://github.com/open-mmlab/mmcv.git
+   cd mmcv
+   MMCV_WITH_OPS=1 pip install -e .  # package mmcv-full, which contains cuda ops, will be installed after this step
+   # pip install -e .  # package mmcv, which contains no cuda ops, will be installed after this step
+   cd ..
+   ```
+
+   **Important**: You need to run pip uninstall mmcv first if you have mmcv installed. Because if mmcv and mmcv-full are both installed, there will be ModuleNotFoundError.
+
+4. Install MMEngine
+
+   ```shell
+   pip install mmengine
+   ```
+
+5. Install MMDetection
+
+   ```shell
+   pip install mmdet
+   ```
+
+   Optionally, you can also build MMDetection from source in case you want to modify the code:
+
+   ```shell
+   git clone https://github.com/open-mmlab/mmdetection.git
+   cd mmdetection
+   pip install -r requirements/build.txt
+   pip install -v -e .  # or "python setup.py develop"
+   ```
+
+6. Clone the MMTracking repository.
+
+   ```shell
+   git clone https://github.com/open-mmlab/mmtracking.git
+   cd mmtracking
+   ```
+
+7. Install build requirements and then install MMTracking.
+
+   ```shell
+   pip install -r requirements/build.txt
+   pip install -v -e .  # or "python setup.py develop"
+   ```
+
+8. Install extra dependencies
+
+- For MOT evaluation (required):
+
+  ```shell
+  pip install git+https://github.com/JonathonLuiten/TrackEval.git
+  ```
+
+- For VOT evaluation (optional)
+  ```shell
+  pip install git+https://github.com/votchallenge/toolkit.git
+  ```
+
+- For LVIS evaluation (optional):
+
+  ```shell
+  pip install git+https://github.com/lvis-dataset/lvis-api.git
+  ```
+
+- For TAO evaluation (optional):
+
+  ```shell
+  pip install git+https://github.com/TAO-Dataset/tao.git
+  ```
+
+Note:
+
+a. Following the above instructions, MMTracking is installed on `dev` mode
+, any local modifications made to the code will take effect without the need to reinstall it.
+
+b. If you would like to use `opencv-python-headless` instead of `opencv-python`,
+you can install it before installing MMCV.
+
+### A from-scratch setup script
```

Comment:
Is mmclassification also needed?

Reply:
We add it in the requirements/runtime.txt